src/map/atcommand.c

```diff
@@ -2744,39 +2744,42 @@ ACMD(guildlevelup)
 	return true;
 }
 
-/*==========================================
+/**
+ * Creates a pet egg in the character's inventory.
  *
- *------------------------------------------*/
+ * @code{.herc}
+ *	@makeegg <pet>
+ * @endcode
+ *
+ */
 ACMD(makeegg)
 {
+	int id;
+	int pet_id;
 	struct item_data *item_data;
-	int id, pet_id;
 
-	if (!*message) {
-		clif->message(fd, msg_fd(fd,1015)); // Please enter a monster/egg name/ID (usage: @makeegg <pet>).
+	if (*message == '\0') {
+		clif->message(fd, msg_fd(fd, 1015)); /// Please enter a monster/egg name/ID (usage: @makeegg <pet>).
 		return false;
 	}
 
-	if ((item_data = itemdb->search_name(message)) != NULL) // for egg name
+	if ((item_data = itemdb->search_name(message)) != NULL) /// For egg name.
 		id = item_data->nameid;
+	else if ((id = mob->db_searchname(message)) != 0) /// For monster name.
+		;
 	else
-		if ((id = mob->db_searchname(message)) != 0) // for monster name
-			;
-		else
-			id = atoi(message);
+		id = atoi(message);
 
-	pet_id = pet->search_petDB_index(id, PET_CLASS);
```

Comment:
I feel having assignment in new line is better and is having better readability . Is there any standard for this?

Reply:
I don't know about a standard for this, but since assignments inside an if-condition are used all over the code, and I like that syntax, I'd prefer this. Just look a few lines above:
`if ((item_data = itemdb->search_name(message)) != NULL)`
`if ((id = mob->db_searchname(message)) != 0)`